scenarios/aws/eks/cluster.go

```diff
@@ -0,0 +1,279 @@
+package eks
+
+import (
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	"github.com/DataDog/test-infra-definitions/components"
+	kubecomp "github.com/DataDog/test-infra-definitions/components/kubernetes"
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	localEks "github.com/DataDog/test-infra-definitions/resources/aws/eks"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Would rename this to `awseks`, at first I thought it was referencing local kind

Reply:
Agree, I just copy pasted what existed, but that looks better to change it to `awseks`

Reply:
Well actually we already have another package named `awseks` refering to `github.com/pulumi/pulumi-aws/sdk/v6/go/aws/eks` . Here `localeks` refer to resource defined in the local repository (`test-infra-definitions`)